.rubocop.yml

```diff
@@ -275,7 +275,7 @@ Metrics/BlockNesting:
 Metrics/ClassLength:
   Description: 'Avoid classes longer than 300 lines of code.'
   Enabled: true
-  Max: 300
+  Max: 350
```

Comment:
Might be better to add an opt out comment on the offending code if it needs to be longer than 300 lines rather than change this globally across the application, by doing this I think we could be making the exception the rule.

Reply:
Done!